src/index.css

```diff
@@ -1,13 +1,27 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
     sans-serif;
```

Comment:
- [ ] For this project, according to the required guidelines, you're expected to use Lato font for your body font. Kindly use the Google Lato font. 👍🏽

<img width="1379" alt="Screenshot 2023-01-14 at 12 18 48 PM" src="https://user-images.githubusercontent.com/50941074/212471474-216cac18-24cb-493a-b3aa-2f45c552215b.png">

Reply:
Thanks for the information